core/aws-lsp-core/src/util/path.ts

```diff
@@ -78,3 +78,16 @@ export function normalize(p: string): string {
     }
     return normalizeSeparator(path.normalize(p))
 }
+
+export function sanitize(inputPath: string): string {
```

Comment:
In aws-toolkit-vscode this santize() was added recently, but I don't get why it was added: https://github.com/aws/aws-toolkit-vscode/pull/6840#discussion_r2018996426

Why is this needed instead of [our normalize()](https://github.com/aws/aws-toolkit-vscode/blob/708f5381c7627d06a6f63552b3f8074672351ed1/packages/core/src/shared/utilities/pathUtils.ts#L92) ?

Reply:
It looks like they do slightly different things that definitely overlap with `normalize`. For example, normalize doesn't remove white space or expand `~`. I agree they definitely overlap and could potentially be refactored to make this more clear.